sound/soc/codecs/max98373-sdw.c

```diff
@@ -774,7 +779,8 @@ static int max98373_sdw_set_tdm_slot(struct snd_soc_dai *dai,
 	struct max98373_priv *max98373 =
 		snd_soc_component_get_drvdata(component);
 
-	if (!tx_mask && !rx_mask && !slots && !slot_width)
+	/* tx_mask is ignored since it's irrelevant for I/V feedback */
+	if (!rx_mask && !slots && !slot_width)
```

Comment:
would it not make sense to print a debug message if someone is trying to set `tx_mask != 0`?

Reply:
good point @lyakh , for rt1308-sdw we used this:

````c
int set_tdm_slot()
{
....
	if (tx_mask)
		return -EINVAL;
...
}
````

so maybe use the same solution for consistency?

@RanderWang maybe you can add this as a fixup! patch, i'd like to merge this to unlock the issue #2169. The additional check is does not add any functional value but is nice to have indeed, a really good candidate for a fixup.

Reply:
done, please check https://github.com/thesofproject/linux/pull/2202